packages/teams-js/src/internal/utils.ts

```diff
@@ -266,3 +266,33 @@ export function callCallbackWithErrorOrResultOrNullFromPromiseAndReturnPromise<T
   });
   return p;
 }
+
+/**
+ * A helper function to add a timeout to an asynchronous operation.
+ *
+ * @param action Action to wrap the timeout around
+ * @param timeoutInMs Timeout period
```

Comment:
>Timeout period [](http://example.com/codeflow?start=22&length=14)

I know it's literally in the name of the variable but perhaps consider adding "in milliseconds" to this description